src/moments.py

```diff
@@ -0,0 +1,95 @@
+import numpy as np
+import scipy as sc
+
+
+class Moments:
```

Comment:
- Move calculations done in 'level_moments' and 'count_moments' into Level class, passing Moments as parameter.
- make Moments a base class for Monomials and FourierFunctions it should define an abstract interface to a set of generalized moments including bounds etc.